ZipBuilder/Sources/ZipBuilder/FirebaseBuilder.swift

```diff
@@ -21,17 +21,15 @@ import Foundation
 struct FirebaseBuilder {
   /// ZipBuilder instance.
   private let zipBuilder: ZipBuilder
-
-  /// Default initializer. If allSDKsPath and currentReleasePath are provided, it will also verify
-  /// that the
   ///
```

Comment:
This doesn't have any documentation other than a parameter label - was this intentional?